methods/secular/Secular_Requirement_Validation.ipynb

```diff
@@ -9,7 +9,8 @@
     "**Original code authored by:** David Bekaert, Heresh Fattahi, Eric Fielding, and Zhang Yunjun \n",
```

Comment:
<p>The HRRR correction is only in ARIA S1-GUNW products for North America, right? NISAR GUNW products will not have HRRR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>